### PR TITLE
Addition of Czech

### PR DESCRIPTION
**Pull request by Lukyn150**
Sorry for the 2 year delay

### CODE REVIEW

File: Czech/RenX_Game.cze [hunk 1]
```diff
@@ -0,0 +1,1598 @@
...
+[Rx_Weapon_VoltAutoRifle]
+ItemName="Voltová Automatická Puška"
+PickupMessage="Voltová Automatická Puška"
+
+[Rx_Weapon_RamjetRifle]
+ItemName="Ramjet Puška"
+PickupMessage="Ramjet Puška"
+
+[Rx_Weapon_Railgun]
+ItemName="Railgun"
```
Comment (SonnyX) [inline Czech/RenX_Game.cze:84]: No Czech word for Railgun?

File: Czech/RenX_Game.cze [hunk 2]
```diff
@@ -0,0 +1,1598 @@
...
+[Rx_DmgType_AATower]
+DeathString="`o se dostal do vzdušného prostoru hráče `k"
+
+[Rx_DmgType_Abduction]
+DeathString="`o se ztratil ve vesmíru"
+MaleSuicide="`o se ztratil ve vesmíru"
+WeaponName="Abducted"
+
+[Rx_DmgType_AGT_MG]
+DeathString="`o was caught inside `k perimeter and was terminated immediately"
```
Comment (SonnyX) [inline Czech/RenX_Game.cze:329]: missing translation

**Comment by SonnyX** [comment]
Seems to be incomplete as of yet, although better than no translation at all ofcourse.

File: Czech/RenX_Game.cze (hunk 2 above)
Comment (SonnyX) [reply Czech/RenX_Game.cze:329]: More missing translations below and above here as well.

File: Czech/RenX_Game.cze (hunk 1 above)
Comment (Lukyn150) [reply Czech/RenX_Game.cze:84]: No, the only translation possible is electromagnetic cannon in Czech but tha's too long.
And some words like skateboard for example don't have a Czech counterpart, English version is used instead.

File: Czech/RenX_Game.cze (hunk 2 above)
Comment (Lukyn150) [reply Czech/RenX_Game.cze:329]: I haven't finished translating the damage types. Are they even in use yet?

**Comment by Lukyn150** [comment]
Also with the current localisation support it doesn't impact that much in the game and the fonts don't support Czech letters.

**Pull request merged**